components/AutoComplete.js

```diff
@@ -0,0 +1,37 @@
+const React = require("react");
+const TextInput = require("ink-text-input").default;
+const { Box, Text, useInput } = require("ink");
+
+const AutoComplete = ({valueControl={val, setValue}, baseList, handleSubmit}) => {
+    const [suggestion, setSuggestion] = React.useState([]);
```

Comment:
`suggestion` defaults as an array but is treated as a string later.

```suggestion
    const [suggestion, setSuggestion] = React.useState('');
```